sdk/storage/Azure.Storage.DataMovement.Blobs/stress/src/Infrastructure/TestSetupHelper.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Test;
+using Azure.Storage.Tests.Shared;
+
+namespace Azure.Storage.DataMovement.Blobs.Stress
+{
+    internal static class TestSetupHelper
+    {
+        public static string Randomize(string prefix = "sample") =>
+            $"{prefix}-{Guid.NewGuid()}";
+
+        public static async Task<Stream> CreateLimitedMemoryStream(
```

Comment:
This feels like something we may have in Common somewhere? Or maybe something that could go in Common? If we even have access to common here. (When I say Common, I mean Shared and cross-compiled in)

Reply:
For me to pull in this method, I would have to pull in literally everything from the Azure.Storage tests. and I did try to do that, and it was much more painful and timeconsuming than doing this

Reply:
Also I just realized it's because these methods when they call "Random" it references the Random Seed in the Recording Test Framework. In stress testing we don't include nor record tests here.

Reply:
Hmmm, seems like the one in Azure.Storage tests is too tightly coupled to the test framework then? Something like this really feels like something that could be a single shared file with no other dependencies. But if that's not the case, this is fine.

Reply:
We still have to figure out a way to not compile in the code that grabs the Random Seed from the Test Recordings. Cause currently today it if a test is recorded and ran in playback it grabs that Random seed from the recording.